### PR TITLE
Remove GA for plausible

### PR DESCRIPTION
**Pull request by ryqndev**
#### Description
Not really utilizing GA. Talked offline with dev team to remove it

### CODE REVIEW

File: src/app/components/Footer/Footer.js [hunk 1]
```diff
@@ -140,18 +140,6 @@ const Footer = () => (
...
-				found{' '}
-				<a
-					target='noreferer'
-					className={cn.mark}
-					href='https://www.google.com/policies/privacy/'
-				>
-					here
-				</a>
-				.
-			</Text>
```
Comment (taesungh) [inline src/app/components/Footer/Footer.js:154]: Question: do we need to include a similar privacy notice when using Plausible?

Comment (ryqndev) [reply src/app/components/Footer/Footer.js:154]: Nope -> main reason why I picked it. It doesn't use cookies or any advanced tracking. https://plausible.io/data-policy

Comment (ryqndev) [reply src/app/components/Footer/Footer.js:154]: Oh also one last thing, They have an API, so we can create some sort of dash tool

File: public/index.html [hunk 2]
```diff
@@ -32,6 +32,7 @@
 	<link rel="preload" href="/static/font/bold.woff2" as="font" type="font/woff" crossorigin>
 	<link rel="preload" href="/static/font/medium.woff2" as="font" type="font/woff" crossorigin>
 
+	<script defer data-domain="designatuci.com" src="https://plausible.io/js/script.js"></script>
```
Comment (taesungh) [inline public/index.html:35]: Question: is specifying `data-domain` enough to avoid having development requests misclassified as actual traffic?

Comment (ryqndev) [reply public/index.html:35]: Yes and no - there is a setting within plausible to turn off localhost traffic.

Comment (ryqndev) [reply public/index.html:35]: It's more of a way to split up dashboards. The data-domain is mostly for this: https://plausible.io/docs/plausible-script#can-i-send-stats-to-multiple-dashboards-at-the-same-time.

**Review by taesungh — APPROVED** [review]
Looks good!

**Pull request merged**